src/exercise/01.js

```diff
@@ -1,29 +1,29 @@
 // useState: greeting
 // http://localhost:3000/isolated/exercise/01.js
 
-import * as React from 'react'
+import React, { useState } from 'react'
 
-function Greeting() {
-  // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+function Greeting({initialName = ""}) {
```

Comment:
If this were on one of our projects, ESLint would complain about spaces within the curly braces and wanting the string value to use single quotes:
```suggestion
function Greeting({ initialName = '' }) {
```

Reply:
👍